src/verilator/VerilatorObject.py

```diff
@@ -0,0 +1,14 @@
+from m5.params import *
+from m5.SimObject import SimObject
+
+class VerilatorObject(SimObject):
+    type = 'VerilatorObject'
+    cxx_header = "verilator/verilator_object.hh"
+
+    cycles = Param.Int(0, "Maximum number of cycles to simulate,
+            0 means unlimited")
+    latency = Param.Latency("Clock period of device under test")
```

Comment:
A better name would be "clockPeriod"...

Actually, if you inherited from "ClockedObject" instead of SimObject, this would come for free (you wouldn't need a parameter). I think that's a better idea.